web/source/build.sh

```diff
@@ -425,24 +425,30 @@ if [ $BUILD_EMBED = true ]; then
         echo
     fi
 
+    if [ $BUILD_DEBUG_EMBED = true ]; then
+        # We currently have an issue with sourcemaps for minified versions in embedded contexts.
+        # We should use the unminified one instead for now.
+        cp $EMBED_OUTPUT_NO_MINI/keyman.js $EMBED_OUTPUT/keyman.js
+        # Copy the sourcemap.
+        cp $EMBED_OUTPUT_NO_MINI/keyman.js.map $EMBED_OUTPUT/keyman.js.map
+        echo Uncompiled embedded application saved as keyman.js
+    fi
+
     if [ $UPLOAD_EMBED_SENTRY = true ]; then
-        pushd $EMBED_OUTPUT
+        if [ $BUILD_DEBUG_EMBED = true ]; then
+            ARTIFACT_FOLDER="release/unminified/embedded"
+            pushd $EMBED_OUTPUT_NO_MINI
+        else
+            ARTIFACT_FOLDER="release/embedded"
+            pushd $EMBED_OUTPUT
+        fi
         echo "Uploading to Sentry..."
-        npm run sentry-cli -- releases files "$SENTRY_RELEASE_VERSION" upload-sourcemaps --strip-common-prefix release/embedded --rewrite --ext js --ext map --ext ts || fail "Sentry upload failed."
+        npm run sentry-cli -- releases files "$SENTRY_RELEASE_VERSION" upload-sourcemaps --strip-common-prefix $ARTIFACT_FOLDER --rewrite --ext js --ext map --ext ts || fail "Sentry upload failed."
```

Comment:
The tweaks here facilitated this local development test:

https://sentry.keyman.com/share/issue/64ae0df206a1467fbc919630cf902e4c/

It was run with a "debug" iOS compilation, so with the unminified embedded code instead of the minified version we publish in our CI builds.